check.go

```diff
@@ -366,7 +370,7 @@ func (c *CheckRunner) handleCheckUpdate(check *api.HealthCheck, status, output s
 	existing.Status = status
 	existing.Output = output
 
-	c.logger.Printf("[INFO] Updating output and status for %q", existing.CheckID)
+	c.logger.Info("Updating output and status for ", "id", hclog.Fmt("%q", existing.CheckID))
```

Comment:
Would you mind removing the extra whitespace after "for"? It also might be helpful for the key name to be "checkId" instead of "id". What do you think?